torch/fx/experimental/proxy_tensor.py

```diff
@@ -821,28 +845,19 @@ def get_torch_dispatch_modes():
 
 
 def get_innermost_proxy_mode():
```

Comment:
Good to eliminate this later

Reply:
looks like it's only used by distributed - agreed, I can probably kill this in a later PR and fix up the distributed code to use the "current" proxy mode.